### ISSUE TITLE
`--no-progress` should suppress benchmark-target's logs

### ISSUE THREAD

**Issue by MahdiBM** [ordo-one/package-benchmark#288]
`--no-progress` should suppress benchmark-target's logs. 
e.g. I see Vapor logs when I run the benchmarks + the comparison in one go, with `benchmark thresholds check`.

**Comment by MahdiBM** [comment]
I think optimally there would be an option to somehow keep the logs for debugging?
Like save the logs somewhere?
So `--no-progress` does suppress the logs, but another flag direct the runner to save the logs to a file.
Something like that ... because the logs might still be needed for debugging.

**Comment by hassila** [comment]
Not sure how I can suppress logs for arbitrary frameworks?

**Comment by MahdiBM** [comment]
I think what I meant by this is that the benchmarking package should simply just suppress all stdout logs of the target when it's running the benchmarks. So if a benchmarks has a bunch of debugging `print`s or other logs, optimally none of them would show up.
Is that hard to do?

**Comment by hassila** [comment]
Nope, fixed in https://github.com/ordo-one/package-benchmark/releases/tag/1.29.1

No rerouting though, if you need to debug simply do not pass --no-progress.

**Comment by MahdiBM** [comment]
sounds good, thanks.
